app/Http/Controllers/ControllerDash.php

```diff
@@ -178,7 +178,9 @@ public function showProfile($year = null, $month = null) {
                 Config::get('scheddy.base').'/api/userSessions/'.$user_id,
                 ['headers' => [
                     'Authorization' => 'Bearer '.Config::get('scheddy.api_key')
-                ]]
+                ],
+                'http_errors' => false
+                ]
```

Comment:
By adding this, the user will have no way to know that there is an issue with scheddy. Below in the if statement, if the status is not 200 then we should return a status message letting the user know something went wrong with scheddy.